<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingTab</class>
 <widget class="QWidget" name="SettingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
border:none;
background-color:transparent;
background:none;
padding:0;
margin:0;
color:#fff;
}



#SettingTab{
background-color:#1f232a;
}
#EditVendor{
background-color:#16191d;
}
QPushButton{
text-align:left;
padding: 5px 10px;

border-top-left-radius:5px;
border-botton-left-radius:5px;
}
QPushButton:pressed{
background-color:grey;
text-align:left;
padding:2px 10px;
color:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;If Database gets Corrupted&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">QFrame{
    border: 2px solid #000000;
border-color:grey;
border-radius:15px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTip">
              <string>Save all default and custom changes</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
  QPushButton {
    border: 2px solid #000000;
border-color:grey;
border-radius:15px;
text-align:center;
}

QPushButton:hover {
    background-color: lightblue;
}
QToolTip {
    color: black; 
    background-color: #FFFFE0; 
}
</string>
             </property>
             <property name="text">
              <string>Save All Changes</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/diskette.png</normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/diskette.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
    border: 2px solid #000000;
border-color:grey;
border-radius:15px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>The maximum number of reports to work on at the same time. If set too high, the UI might freeze while fetching reports but the fetching process will continue.</string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>Concurrent reports</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>The number of seconds the program will allow a vendor to respond to each report request before cancelling it.</string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>Request timeout</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>The number of seconds the program will wait between sending each report request to a given vendor.</string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>Report request interval</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>Can change if you find the load on your desktop is too high</string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>Concurrent vendors</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>How program identifies itself to the SUSHI servers. Some vendors will reject some particular user agents. Only change this if there is a known problem as it will affect all requests to all vendors. </string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>User agent</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>We can set where to save search database, vendor data file and yearly reports here</string>
             </property>
             <property name="styleSheet">
              <string notr="true">Border:none;</string>
             </property>
             <property name="text">
              <string>Data folders directories</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="3">
            <widget class="QSpinBox" name="spinBox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QSpinBox {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QSpinBox" name="spinBox_2">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QSpinBox {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QSpinBox" name="spinBox_3">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QSpinBox {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEdit">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QSpinBox" name="spinBox_4">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QSpinBox {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    color: black; /* 文本颜色为黑色 */
	background-color: #cccccc;
}
</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="pushButton_2">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover {
    background-color: #0000FF; /* 鼠标悬停时的背景颜色为蓝色 */
    color: #FFFFFF; /* 鼠标悬停时的文字颜色为白色 */
}
</string>
             </property>
             <property name="text">
              <string>Choose</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/folder.png</normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/folder.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="comboBox">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox {
    color: black; /* QComboBox中的文本颜色为黑色 */
	background-color: #cccccc;
}

QComboBox QAbstractItemView {
    color: black; /* 下拉列表中的文本颜色为黑色 */
}
</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Search database</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vendor data file</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yearly reports</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other reports</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">QFrame{
    border: 2px solid #000000;
border-color:grey;
border-radius:15px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <family>Georgia</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTip">
              <string>Recreate or regenerate the database that stores information related to the usage statisitics. when you want to do data consistency, data integrity, update and changes.</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
  QPushButton {
    border: 2px solid #000000;
border-color:grey;
border-radius:15px;
text-align:center;
}

QPushButton:hover {
    background-color: lightblue;
}

QToolTip {
    color: black; 
    background-color: #FFFFE0; 
}

</string>
             </property>
             <property name="text">
              <string>Rebuild Database</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/database.png</normaloff>../Final Project/COUNTER-Release-5.1/ui/resources/Icons/database.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Final Project/COUNTER-Release-5.1/ui/resources.qrc"/>
 </resources>
 <connections/>
</ui>
